databricks/koalas/frame.py

```diff
@@ -4226,7 +4226,13 @@ def to_pandas(self):
         return self._internal.to_pandas_frame.copy()
 
     # Alias to maintain backward compatibility with Spark
-    toPandas = to_pandas
+    def toPandas(self):
+        warnings.warn(
+            "DataFrame.toPandas is deprecated as of DataFrame.to_pandas. "
+            "Please use the API instead.",
+            FutureWarning,
```

Comment:
`DeprecationWarning`?

Reply:
Also `toPandas.__doc__ = to_pandas.__doc__` is needed?

Reply:
> `DeprecationWarning`?

Oh, then maybe should we fix the other `FutureWarning`s actually related with deprecation to the `DeprecationWarning` ??

Reply:
Oh, I used `FutureWarning` instead of `DeprecationWarning`. The semantic of both were changed in newer Python versions. The former implies a deprecation to end users (https://docs.python.org/3/library/exceptions.html#FutureWarning). The latter is for developers (https://docs.python.org/3/library/exceptions.html#DeprecationWarning). `DeprecationWarning` is ignored by default.

Reply:
ah, I see, then `FutureWarning` would be better.

Reply:
Thanks for the comments !